friends/src/Components/FriendsList.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+// import axios from 'axios';
+import { Route } from 'react-router-dom';
+
+import Friend from './Friend/Friend';
+import FriendForm from './FriendForm/FriendForm';
+
+class FriendsList extends React.Component {
+    constructor(props){
+        super(props)
+        this.state = {
+            friends:[]
+        }
+    }
+
+    render() {
+        return ( 
+            <div className="list-container">
+                {this.props.friends.map(friend=>{
+                    return <div key={friend.id}>
+                        <Friend friend={friend} deleteFriend={this.props.deleteFriend}/>
+                        {/* Is the pattern below an anti-pattern? */}
```

Comment:
No, this isn't an anti pattern. Is there a specific reason that you thought it was?

Reply:
I wasn't sure if it was bad practice to set a route based off of a template literal because using a route is kind of like conditional rendering itself.